### DISCUSSION TITLE
Welcome to designaire-z390-opencore Discussions!

### DISCUSSION THREAD

**Discussion by baughmann** [baughmann/designaire-z390-intel-i9-9900k-opencore#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by seven-of-eleven** [comment]
# OC 0.8.7 Update
Just to let everyone know I plan to update over the next week.

**Comment by seven-of-eleven** [comment]
## OC 0.9.6 Update
FYI. I'm waiting for the next version of OC before updating and will include some notes on Sonoma as well.

- Fenvi / BRCM wifi cards have reported issues with Sonoma. There is work arounds but I would hold off updating at this point unless you're supper eager to get on the latest OS. Otherwise everything I've tested seems to be working.

All the best,

**Comment by mark-in-seattle** [comment]
Thank you very much for all your hard work to support those of us using the Gigabyte Z390 Designare motherboard in our Hackintosh.  Mine currently under Opencore v0.8.8 (4 years ago began with OC v0.5.6 ?).

Came to your site because, my case front panel power-ON switch failed.  Symptom was standby powersupply lit up the motherboard LED's etc, however pushing the momentary ON switch did nothing, ... system did not boot.  Mistakenly tried resetting the CMOS RAM first because my Fluke DVM said the button battery was 3.01 vdc, maybe a little low and unstable.  Replaced with newer battery and cleared CMOS.  Soldered in a replacement momentary power-ON switch at the case front panel and booted the system into Z390 BIOS.

Manually reconfigured my v9b BIOS, however this older BIOS (2019) did not provide a CFG-unlock option, meaning I would have to remember how to enable OpenShell and manually unlock the CFG bit flag.  Surprisingly OC v0.8.8  may have unlocked it with a kext(?) because I was able to boot into my Monterey v12.7.2 successfully.  Even my ZFS pools were visible.

Have heard that a Hackintosh will be more stable if the CFG flag is unlocked directly in the BIOS, so found your site researching how to do this.  My OS is backed up with Carbon Copy Cloner, and my data to independent NAS's (also off-site every other month).  With these backups in place I will use your OC v0.9.6  EFI files after updating the Z390 Designare to BIOS v9.  Very excited to see if Thunderbolt will work and more of the USB3.2 ports.  My earlier USB port mapping was admittedly not very rigorous so only a couple USB3 ports worked and no Thunderbolt.

My project to "pay it forward" in retirement is working on an affordable residential kit-home design with a target cost of $100/psf.  After 3 years effort using a prefab modular approach I wasn't smart enough to meet my target cost, so I went in a different direction in home design: structural steel portal frames as a skeleton with panelized floors and exterior elements.  Reduced the 2,000-sf parts count by 80% and I think the construction time will be 6 weeks total from a 4 person crew.  A SkyCiv FEA cloud subscription provides enough stress analysis to give me confidence that submitting the Sketchup drawings to a local structural engineering firm for code review won't be overly embarrassing.

Stay safe.  Many regards from Seattle, WA in NW corner of United States.
-- mark early

**Comment by mark-in-seattle** [comment]
Question regarding a missing MacOS recovery boot option which my earlier OC-0.8.8 config.plist offered during boot up.  Successfully applied your OC-0.9.6 config.plist with my own earlier SMBIOS values (iMac19,1  ,ROM, SystemSerialNumber ..etc).  Tested the new OC-0.9.6 in the EFI partition of a bootable USB memory stick, then copied the new OC-0.9.6 EFI folder to the EFI partition of my main NVME SSD boot drive.   Booted fine, though earlier USB testing indicated I needed to move my wireless USB keyboard link device from USB2 ports at the top of the Z390-Designare motherboard backplane to USB3 ports further down.
 
My earlier OC-0.8.8 displayed 3 icon boot options:     ---- EFI(?)----    ----MacHD-----     ----Recovery 12.7.2----
With OC-0.9.6    there is only a single icon boot option:   ----MacHD----       (with very little time delay before defaulting to it as selected)

Can you point me to a document or discussion about adding the MacOS  "Recovery boot option" for this new version of OC ?

Before moving to the new OC-0.9.6 I successfully updated the Z390-Designare to BIOS-v9 and loaded your BIOS-settings file.  Double checked the BIOS config and documented, all looked good.  However, I was not able to invoke the boot menu using F12.  Always had to hit DEL key, enter the BIOS config screen and modify Boot options manually, then reboot.

The MacOS (Monterey v12.7.2) About-this-Mac --> "System Report" --> Thunderbolt/USB4  tab is empty "No drivers are loaded".  I'm assuming this is because the kexts enabling Thunderbolt on a Hackintosh are not recogized by the MacOS as such.  My external thunderbolt drive is readable when plugged in, however I think it is also USB3.2 compatible.  Not sure how to test for a genuine thunderbolt link.

**Reply by seven-of-eleven** [reply]
Happy to hear your making good progress with the OpenCore update. Newer versions of OpenCore have the utilities hidden by default. You can tap the space bar when at the OpenCore menu to show the hidden items or you can change the default behaviour. From memory I think the settings is in the MISC section of the config.plist file.

That's odd about the F12 not working. It may be a keyboard issue? If your keyboard has a function key try holding that down while pressing F12.

The Z390 Designare has Thunderbolt 3 controller with "2 x USB Type-C™ ports on the back panel, with USB 3.1 Gen 2 support" (from the Gigabyte website). I can't recall from memory where they show up in the System Report.  I can check and circle back to you if needed. I borrowed a Thunderbolt device and to test it was working, but I don't use it regularly so YMMV. If the device also has USB support I'm not sure either about how you would test the connections.

Best of luck.

**Reply by mark-in-seattle** [reply]
Thank you very much for your helpful reply.

Will follow the suggestion about hitting SPACE-BAR to display other boot options and also look in the config.plist to change default behavior and then report back here to help other users who might google this thread looking for answers to address similar issues. 

Indeed my F12 key not bringing up boot menu during boot sequence is rather strange, since the DEL key works fine to trigger BIOS menu.   My keyboard is a Logitech wireless "MX-Keys" version, which otherwise seems to work fine.  Probably some quirk with Logitech's implementation of the pre-OS keyboard comm protocol.

Thunderbolt 3 compatibility: will do my best to report back for the benefit of other Z390-Designare owners regarding Thunderbolt 3 ports.  I have only tested with external cases holding PCIE-4 NVME-m2 memory sticks that are specifically TB3, TB4 and as you mention USB3.1 Gen 2 compatible.  So with the MacOS "system info report" not indicating if Thunberbolt is active, it is difficult to determine if the USB-C ports enabled on the Z390-Designare are using the thunderbolt protocol or USB3.x.   If time allows I will do a data transfer speedtest to collect more info about the likelyhood of thunderbolt involvement on these two USB-C ports.

Thank you again for creating a very useful site.

**Comment by mgreissd** [comment]
Thank you Baughmann for one of the most concise and comprehensive Z390 Designare hackintosh repositories.   I've been struggling for months to try to get my audio working and your style and clarity made me want to give this "build" a try.  I've been running my Designare since before the Mac Pro came out.   I have only one problem and cannot get it resolved:  Audio is not working properly for Analog Outputs (Green 1/8" mini jack).  Before this build, I had a high volume thump, and then silence for a while.  With your build, I have a little click, then the audio seems to reset after about 1- 1.5 seconds.  Just a thump, no actual OS sounds.

That being said, following your comprehensive guide has given me a much faster machine as far as screen draws go.  Just checked... wifi and bluetooth are working.  Even networking is a lot faster than before. 

If anyone is still checking in here once in a while, I would love some pointers on how to resolve the audio.  It seems like a bit of overkill to run the audio out through the MOTU 828es USB, and a bit of underkill to run it through HDMI to my TV monitor.

Wow this thing is fast.  Thanks so much.

**Reply by baughmann** [reply]
@mgreissd thank you for the kind words, glad you're enjoying it. Be sure to direct your thanks to @seven-of-eleven though, as he is the new maintainer and has put in even more work than myself

**Reply by mgreissd** [reply]
By all means, much thanks to @seven-of-eleven  for maintaining this and making my system run so much faster.  I'm beginning to think my lack of audio outputs through the audio jacks might be a hardware problem, but at this point I'm not sure where to go to resolve the issue.

**Comment by mgreissd** [comment]
I'm not sure if this is the right place, but I seem to be having a problem with audio that could be solved with SSDT-HPET.  The onboard analog sound locks up temporarily - never plays a sound.  I bought a cheap ($7) USB to audio I/O adapter and if I use that for sound then everything works fine.  If I try to play through the motherboard built in speakers (11) then the sound goes thump and after a few seconds will there's another thump as things "reset."

Unfortunately I don't have a way to create the SSDT-HPET because I can't get SSDTTime to generate one - I need to dump the DSDT and you can't do that in OSX.
 
 Does any of this make sense?
